foodsaving/conversations/tasks.py

```diff
@@ -34,10 +46,15 @@ def notify_participants(message):
             user__groupmembership__group=message.conversation.target,
         )
 
-    for participant in participants_to_notify:
+    for participant in participants_to_notify.distinct():
+        messages = participant.unseen_and_unnotified_messages().all()
```

Comment:
Looks like we didn't filter for participants on their notified_up_to, so could return 0 messages here? (would be the case if this is run in quick succession I think)

Reply:
I added a filter for notified_up_to. I guess it still could happen that a messages is being marked as seen while the task is running. This might result in 0 messages, so I could add another filter... or just wait if this happens in production. It will throw an error, get reported by sentry and not affect anybody..